src/components/segmentedControl/segment.tsx

```diff
@@ -55,6 +55,7 @@ export type SegmentProps = SegmentedControlItemProps & {
    */
   style?: StyleProp<ViewStyle>;
   testID?: string;
+  iconTintColor?: string;
```

Comment:
New prop should be added to api file as well

Reply:
You have the 'tintColor' in the SegmentedControlItemProps's 'iconStyle'. And even if you want to add this prop I think it makes more sense to add it there

Reply:
I guess I can expose it.